### DISCUSSION TITLE
Certificate error when using a private docker registry

### DISCUSSION THREAD

**Discussion by lejsekt** [microsoft/vscode-docker#3351] [Q&A] [answered]
Hi,
this is a possible duplicate of https://github.com/microsoft/vscode-docker/issues/3255, but the suggested answer to use VS Code Insiders is not working for me.

We have an on prem docker registry behind https (let's encrypt certificate). If I try to connect to the registry from VS Code using the docker extension, it says that the certificate has expired. Docker pull etc. from the terminal is working and if I check the certificate using openssl, it is indeed valid.

**Comment by bwateratmsft** [comment, marked as answer]
Yeah, it seems the issue came back: https://github.com/microsoft/vscode/issues/136787

It's all the way upstream in Electron so it's far out of our control. There is a user-suggested [workaround](https://github.com/microsoft/vscode/issues/136787#issuecomment-968771447) of applying setting `"http.systemCertificates": false`; does that work if you set it?

**Reply by lejsekt** [reply]
Yes, that works. Thank you.
